councilmatic_core/models.py

```diff
@@ -243,6 +247,8 @@ class Meta:
         on_delete=models.CASCADE,
     )
 
+    shape = JSONField()
```

Comment:
In the `0.x` line, `Post.shape` was stored as a `TextField` and data was de/serialized by hand. I'm assuming this is because `0.x` was written before Django and Postgres added full support for `JSONField`, so I went ahead and adjusted it.

Reply:
I don't know if it's better, but the database is postgis enabled if there would be any facilities we wanted to take advantage of there.